django/thunderstore/api/cyberstorm/views/user.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth import get_user_model
+from django.http import Http404, HttpRequest, HttpResponse
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from thunderstore.api.cyberstorm.serializers import CyberstormUserSerializer
+
+User = get_user_model()
+
+
+class UserAPIView(APIView):
+    permission_classes = []
+
+    @swagger_auto_schema(
+        responses={200: CyberstormUserSerializer()},
+        operation_id="api.user",
+    )
+    def get(self, request: HttpRequest, user_identifier: str) -> HttpResponse:
+        try:
+            u = User.objects.get(
+                username=user_identifier,
+            )
+        except User.DoesNotExist:
+            raise Http404(f"Coudln't find user with the identifier {user_identifier}")
```

Comment:
Use `get_object_or_404` imported from the rest framework if you implement this yourself.